emqx/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fa40ec7e-e8f6-4c4b-a675-31716b23a9df",
+  "app_id": "emqx",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Emqx integration with openmetrics v2",
```

Comment:
Can you update this description to briefly describe what the integration is sending into DD?

Reply:
@zhongwencool There are still a few unresolved comments on this file - please address these as well. Thank you!